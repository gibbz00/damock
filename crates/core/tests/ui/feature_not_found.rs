#[derive(mock_default::Mock)]
#[mock(feature = "mocks")]
enum Foo {
    #[mock]
    Bar,
}

fn main() {
    compile_error!("")
}
