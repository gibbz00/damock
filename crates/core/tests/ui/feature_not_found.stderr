error:
 --> tests/ui/feature_not_found.rs:9:5
  |
9 |     compile_error!("")
  |     ^^^^^^^^^^^^^^^^^^

warning: unexpected `cfg` condition value: `mocks`
 --> tests/ui/feature_not_found.rs:2:8
  |
2 | #[mock(feature = "mocks")]
  |        ^^^^^^^^^^^^^^^^^ help: remove the condition
  |
  = note: no expected values for `feature`
  = help: consider adding `mocks` as a feature in `Cargo.toml`
  = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration
  = note: `#[warn(unexpected_cfgs)]` on by default
